kratos/spatial_containers/geometrical_objects_bins.h

```diff
@@ -51,6 +51,12 @@ class KRATOS_API(KRATOS_CORE) GeometricalObjectsBins
     /// Pointer definition of GeometricalObjectsBins
     KRATOS_CLASS_POINTER_DEFINITION(GeometricalObjectsBins);
 
+    /// The point type definition
+    using PointType = Point;
```

Comment:
I will approve, but I don't understand. Why do you want to make this type an alias, if you are hard-coding it here? 

I assume it is because you are going to use this as a base of another bins?

Reply:
Basically that